diff -uwNr iphone7/WebKit/Source/WebInspectorUI/UserInterface/Base/Main.js /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Base/Main.js
--- iphone7/WebKit/Source/WebInspectorUI/UserInterface/Base/Main.js	2024-08-12 06:24:18.593136994 -0700
+++ /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Base/Main.js	2024-08-11 15:30:08.396373539 -0700
@@ -55,8 +55,7 @@
 
 WI.NarrowLayoutMaximumWidth = 700;
 
-WI.loaded = function()
-{
+WI.loaded = function() { WI._initializeWebSocketIfNeeded();
     if (InspectorFrontendHost.connect)
         InspectorFrontendHost.connect();
 
diff -uwNr iphone7/WebKit/Source/WebInspectorUI/UserInterface/Base/Utilities.js /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Base/Utilities.js
--- iphone7/WebKit/Source/WebInspectorUI/UserInterface/Base/Utilities.js	2024-08-12 06:24:18.685138142 -0700
+++ /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Base/Utilities.js	2024-08-11 15:30:08.436374038 -0700
@@ -375,7 +375,7 @@
                     }
 
                     if (node.nodeType === Node.TEXT_NODE) {
-                        let start = node === this ? offset - 1 : node.nodeValue.length - 1;
+                    let start = node === this ? (offset - 1) : (node.nodeValue.length - 1);
                         for (var i = start; i >= 0; --i) {
                             if (stopCharacters.indexOf(node.nodeValue[i]) !== -1) {
                                 startNode = node;
@@ -626,10 +626,6 @@
                 return false;
 
             let constructor = array.constructor;
-            // ugly hack to fix Chrome/Chromium problem
-            if (typeof window.Float16Array === "undefined") {
-                window.Float16Array = Float32Array; // prevent ReferenceError
-            }
             return constructor === Int8Array
             || constructor === Int16Array
             || constructor === Int32Array
@@ -637,7 +633,6 @@
             || constructor === Uint8ClampedArray
             || constructor === Uint16Array
             || constructor === Uint32Array
-            || constructor === Float16Array
             || constructor === Float32Array
             || constructor === Float64Array;
         }
@@ -1052,7 +1047,7 @@
                     // If the number is greater than zero and ends with a "$",
                     // then this is a substitution index.
                     if (number > 0 && format[index] === "$") {
-                        substitutionIndex = number - 1;
+                    substitutionIndex = (number - 1);
                         ++index;
                     }
                 }
diff -uwNr iphone7/WebKit/Source/WebInspectorUI/UserInterface/Main.html /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Main.html
--- iphone7/WebKit/Source/WebInspectorUI/UserInterface/Main.html	2024-08-12 06:24:19.293145725 -0700
+++ /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Main.html	2024-08-11 15:30:08.388373439 -0700
@@ -318,8 +318,7 @@
     <script src="External/three.js/three.js"></script>
     <script src="External/three.js/OrbitControls.js"></script>
 
-    <script src="Base/WebInspector.js"></script>
-    <script src="Base/BrowserInspectorFrontendHost.js"></script>
+    <script src="Base/WebInspector.js"></script><script src="InspectorFrontendHostStub.js"></script><link rel="stylesheet" href="AdditionalStyle.css">    <script src="Base/BrowserInspectorFrontendHost.js"></script>
     <script src="Base/Platform.js"></script>
     <script src="Base/Debouncer.js"></script>
     <script src="Base/DebuggableType.js"></script>
diff -uwNr iphone7/WebKit/Source/WebInspectorUI/UserInterface/Protocol/Connection.js /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Protocol/Connection.js
--- iphone7/WebKit/Source/WebInspectorUI/UserInterface/Protocol/Connection.js	2024-08-12 06:24:19.413147222 -0700
+++ /home/jcomeau/tmp/iphone7.last_known_good/WebKit/Source/WebInspectorUI/UserInterface/Protocol/Connection.js	2024-08-11 15:30:06.352348038 -0700
@@ -72,7 +72,7 @@
     dispatch(message)
     {
         let messageObject = typeof message === "string" ? JSON.parse(message) : message;
-        console.debug("Connection.dispatch: messageObject: " + message);
+
         if ("id" in messageObject)
             this._dispatchResponse(messageObject);
         else
